frontend/src/components/DialogApp/DialogApp.tsx

```diff
@@ -48,24 +56,30 @@ export const DialogApp: React.FC<{
   return (
     <>
       <Dialog open={openApp} onClose={closeAll} maxWidth="sm" fullWidth>
-        <DialogTitle >{title}</DialogTitle>
+        <DialogTitle>{title}</DialogTitle>
         <DialogContent>
           <DialogContentText>
-            You can use a browser-based {type} launcher to access this connection.
-            Alternatively, you can download and install {type}
+            You can use a browser-based {type} launcher to access this connection. Alternatively, you can download and
+            install PuTTy
```

Comment:
I was using `type` becouse there are more options than Putty

Reply:
Ah this was a merge conflict- not intentional - good catch